docs/install/install-redis/install-redis-on-windows.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Install Redis on Windows"
+title: "Install Valkey on Windows"
 linkTitle: "Windows"
 weight: 1
-description: Use Redis on Windows for development
+description: Use Valkey on Windows for development
 aliases:
 - /docs/getting-started/installation/install-redis-on-windows/
 ---
 
-Redis is not officially supported on Windows. However, you can install Redis on Windows for development by following the instructions below.
+Valkey is not officially supported on Windows. However, you can install Valkey on Windows for development by following the instructions below.
 
-To install Redis on Windows, you'll first need to enable [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install) (Windows Subsystem for Linux). WSL2 lets you run Linux binaries natively on Windows. For this method to work, you'll need to be running Windows 10 version 2004 and higher or Windows 11.
+To install Valkey on Windows, you'll first need to enable [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install) (Windows Subsystem for Linux). WSL2 lets you run Linux binaries natively on Windows. For this method to work, you'll need to be running Windows 10 version 2004 and higher or Windows 11.
 
 ## Install or enable WSL2
 
 Microsoft provides [detailed instructions for installing WSL](https://docs.microsoft.com/en-us/windows/wsl/install). Follow these instructions, and take note of the default Linux distribution it installs. This guide assumes Ubuntu.
 
-## Install Redis
+## Install Valkey
```

Comment:
No debian yet, so this section is not accurate.

Reply:
OK, I'll delete the section.